source/jobs/prerequisites/oracle.txt

```diff
@@ -10,13 +10,13 @@ Configure Migration Prerequisites for Oracle
    :depth: 1
    :class: singlecol
 
-To run migration jobs from an Oracle source database, the database may require 
-some configuration changes. If Relational Migrator determines the 
+When migrating data from an Oracle source database, the database may require 
```

Comment:
[question] Is the subject the oracle database or the mongodb database?

```suggestion
When you migrate data from an Oracle source database, that database may require 
```

Reply:
Oracle, ty for the catch